docs/src/modules/java/pages/agents.adoc

```diff
@@ -432,8 +432,7 @@ available tools from each server based on their name.
 It is also possible to intercept, modify, or deny MCP tool requests, or their responses by defining a `RemoteMcpTools.ToolInterceptor`.
 
 [source,java]
-.{sample-base-url}/doc-snippets/src/main/java/com/example/application/RemoteMcpWeatherAgent.java[RemoteMcpAgent
-.java]
```

Comment:
Intersting. There was a breakline here which I removed. 

I know it fails saying that `RemoteMcpAgent.java` does not match `RemoteMcpWeatherAgent.java`. So, with the breakline it check was skipped. 

Is that part of our Antora template?

Reply:
We have some validation of that in CI.

What I have done is to not use the linked `{sample-base-url}` in those cases, but just use
```
.RemoteMcpAgent.java
```

Reply:
but here it was a valid validation error, I see